plugins/net_plugin/net_plugin.cpp

```diff
@@ -537,7 +537,7 @@ namespace eosio {
 
       void on_accepted_block_header( const signed_block_ptr& block, const block_id_type& id );
       void on_accepted_block( const signed_block_ptr& block, const block_id_type& id );
-      void on_voted_block( uint32_t connection_id, vote_status stauts, const vote_message_ptr& vote );
+      void broadcast_vote_message( uint32_t connection_id, vote_status stauts, const vote_message_ptr& vote );
```

Comment:
Because the same function is used for two different signals, a `on_` name probably isn't the best choice. I think it makes more sense here to use a name describing what the function does, rather than what signals it responds to.